InterfaceAdapters/Controllers/DotLanches.Controllers/AdapterPagamentoController.cs

```diff
@@ -20,13 +20,31 @@ public AdapterPagamentoController(IPagamentoRepository pagamentoRepository, IPed
             _checkout = checkout;
         }
 
-        public async Task<PagamentoViewModel?> ProcessPagamento(int idPedido, int queueKey)
+        public async Task<String> ProcessPagamento(int idPedido)
```

Comment:
Precisamos ajustar para refletir o que o método faz realmente:
```suggestion
        public async Task<String> RequestPagamentoQRCode(int idPedido)
```